backends/tc/ebpfCodeGen.cpp

```diff
@@ -408,13 +408,171 @@ void TCIngressPipelinePNA::emitLocalVariables(EBPF::CodeBuilder *builder) {
 }
 
 // =====================EBPFPnaParser=============================
+EBPFPnaParser::EBPFPnaParser(const EBPF::EBPFProgram *program, const IR::ParserBlock *block,
```

Comment:
Please mention the source of this code in comments. as some part is copied from ebpf

Reply:
addressed